src/open_inwoner/accounts/admin.py

```diff
@@ -23,7 +24,14 @@ class _UserChangeForm(UserChangeForm):
     def clean(self, *args, **kwargs):
         cleaned_data = super().clean(*args, **kwargs)
 
+        if (
+            cleaned_data.get("photo")
+            and cleaned_data.get("contact_type") != ContactTypeChoices.begeleider
+        ):
+            raise ValidationError(_("Only a 'begeleider' user can add a photo."))
```

Comment:
I wouldn't limit this to 'begeleider' only, even though it might be requested from a viewpoint of a begeleider if another type of user wants to upload a foto I'd suggest to allow this.

Of course this does open up possible 'uncomfortable' situations, so I'll add a task to double-check this.

Reply:
Yes, I went only for begeleiders because it's the phrase that the user story starts with. I have based task 1191 on this branch as well, should I wait?

Reply:
No need to wait, lets continue with the assumption that this is begeleider-only. 1192 is open to check this assumption.